airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryAsyncFlush.java

```diff
@@ -48,7 +48,7 @@ public void flush(final StreamDescriptor decs, final Stream<PartialAirbyteMessag
 
       stream.forEach(record -> {
         try {
-          writer.accept(record.getSerialized(), record.getRecord().getEmittedAt());
+          writer.accept(record.getSerialized(), Jsons.serialize(record.getRecord().getMeta()), record.getRecord().getEmittedAt());
```

Comment:
@gisripa I think this is what I need to do, but I'd like you to confirm that

Reply:
looks correct to me. One thing to verify is if BigQuery is ok adding extra stuff in the CSV file, we had to pass a flag in snowflake to ignore this `meta` information until we add the migration code to populate `meta` in raw table.

Reply:
I think that my addition of V2_WITH_META means that this value will be ignored when writing the CSV

Reply:
yeah looks like it. seems ok, when we introduce meta in BQ this will work.